app/src/main/java/com/yeonae/chamelezone/ext/ViewExt.kt

```diff
@@ -20,3 +22,11 @@ fun ImageView.glideImageSet(image: Int, width: Int, height: Int) {
         .centerCrop()
         .into(this)
 }
+
+fun glideImageUriSet(view: Activity, image: Uri, width: Int, height: Int, imageView: ImageView) {
```

Comment:
```suggestion
fun ImageView.glideImageUriSet(image: Uri, width: Int, height: Int) {
```
activity를 인자로 받을 필요는 없어보여요

Reply:
수정했습니다